data/src/main/java/cmm/apps/esmorga/data/event/datasource/EventDatasource.kt

```diff
@@ -13,4 +13,5 @@ interface EventDatasource {
         throw EsmorgaException(message = "Unsupported operation", source = Source.UNSUPPORTED, code = ErrorCodes.UNSUPPORTED_OPERATION)
     }
 
+    suspend fun getEventById(eventId: String): EventDataModel
```

Comment:
Would be better to align and either use default implementation to throw Unsupported Exceptions always or never.